docs/minimal/transformations.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: default
+title: Custom Transformation
+parent: Minimal downtime migrations
+nav_order: 4
+---
+
+# Minimal downtime migrations for MySQL
+{: .no_toc }
+
+For cases where a user wants to handle a custom transformation logic, they need to specify the following parameters in the [Datastream To Spanner](https://github.com/GoogleCloudPlatform/DataflowTemplates/tree/main/v2/datastream-to-spanner) template - a GCS path that points to a custom jar, fully classified custom class name of the class containing custom transformation logic and custom parameters which might be used by the jar to invoke custom logic to perform transformation.
+
+Steps to perfrom customization:
+1. Implement custom transformation logic for forward migration in the [toSpannerRow](https://github.com/GoogleCloudPlatform/DataflowTemplates/blob/main/v2/spanner-custom-shard/src/main/java/com/custom/CustomTransformationFetcher.java#L42) method of the **CustomTransformationFetcher.java**. Details of the MigrationTransformationRequest class can be found [here](https://github.com/GoogleCloudPlatform/DataflowTemplates/blob/main/v2/spanner-migrations-sdk/src/main/java/com/google/cloud/teleport/v2/spanner/utils/MigrationTransformationRequest.java).
```

Comment:
Explicitly call out implementing toSourceRow() may be skipped if not using reverse replication?